src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/InternalSyntax/SyntaxToken.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: is this a BOM that was missing, or now missing a BOM?

Reply:
Not sure! I'll take a look.

Reply:
Looks like the BOM is now present:

![image](https://github.com/user-attachments/assets/d079a522-70fe-48af-bca8-55b0d8389ee2)